content/sites/sites.dcf

```diff
@@ -420,6 +428,15 @@ Notes: good first mountain.  Beware wind over the back.
 SeeAlso: [HangGliding.org](https://www.hanggliding.org/wiki/Slide_Mountain)
 
 
+Name: Sylmar
+State: CA
+LaunchGPS: 'Kagel Mtn'=(34.333230, -118.384430, 3500), 'Towers'=(34.342452, -118.406573, 3600)
+LZGPS: 'Main LZ'=(34.317404, -118.406591, 1400)
+Requirements: H3
+SeeAlso: [Sylmar HG Association] (http://www.shga.com/)
```

Comment:
Can you please remove the space between `[...]` and `(...)`?  Otherwise, I don't think it'll be recognized as a Markdown link.